src/en/space-station-14/departments/service/proposals/gnome-ghost-role.md

```diff
@@ -0,0 +1,85 @@
+Do YOU need someone to save you when the air is poisoned but nobody can help?
+Do _YOU_ need something to do once your soul is free floating and ready for harvest?
+
+Never fear! Nanotransen Reaserch and Devopment brings you a breakthrough in both biotech and anomaly studies! 
+THE GNOME! The best helper that botany can ask for! being small enough to get into those TIGHT spaces to do
+tough jobs we gaurentee you will be happy* for their help.
+
+(*Any defects are the responsibility of the station botanist)
+(rumors of gnomes commiting Homicide are false and defamitory. There is no Gnome king)
+
+# Overview
+This design document adds a crafty and fragile ghost role tied to botany.
+
+## Gnome features
+
+ They are a ghost role For early to late round use,
+ they have the following abilities and traits:
+ - A hand slot to pick up items
+ - they can drag items for storage (such as a survival box or backpack)
+ - they can be picked up and worn as hats and die when thrown
+ - they are healed and revived with glue
+ - Gnomes are grown from a Gnome hat that can be planted into a gnome plant that grows the gnome
+ - they are very fragile (10 health) but dont need air and are resistant to tempature damage 
+ - being hollow they are weak to pressure changes that cause them to break quickly, space is an instant death.
+ - can be gibbed into a gnome shard and glass to plant another gnome plant
+ - has a gnome accent that replaces all "no" at the start of words to "GNO" and certain past tence words to GNOMED eg: killed > GNOMED (The captain is GNOMED)
+ - weak to blunt, slightly resistant to slash and peircing, and very resistant to burns
+ - rooms can be "Gnome proofed" with mouse traps
+ - Gnomes act like a normal garden gnome untill possesed by a ghost player
+
+## Gnome Design And intent 
+
+Gnomes are meant to be a way for players to create more ghost roles for dead players and offer new ways to help the station. 
+The intent is for Gnomes to have good utility that leans twords being helpfull to the station and to add more variety of content to botany players. 
+Gnomes while intended to be happy go lucky helpers have a very large potential for petty crime, this is balanced out in three ways:
+ - very low health, if caught annoying crew or stealing or other activities they can be easily killed by whatever is on hand 
+ - no inventory, everything must be held and containers cannot be dragged under doors meaning major theft has to be planned in advance 
+ - cannot use guns 
+
+Gnomes also may get annoying and demand server resourses if there are enough of them. currently 6 gnomes is the ideal normal max, although if botany tries very very 
+hard they may make more than six. the idea is one Gnome per hat. being this limited however it would be simple for a greytide to round remove all of the gnomes. 
+gnomes spawning a plantable gnome shard on death helps prevent gnomes from being removed from the round entirely.
+The gnome hat will spawn as maints loot. 
```

Comment:
The amount of gnomes should be decided by each mapper or by the amount of seeds in the botany vendor/locker, and the seeds shouldn't be multipliable in any way